src/form.schemas.tsx

```diff
@@ -352,3 +373,45 @@ export const CatalogueCategorySchema = z.object({
     .superRefine((data, ctx) => checkForDuplicatePropertyNames(data, ctx))
     .optional(),
 });
+
+// ------------------------------------ CATALOGUE ITEMS ------------------------------------
+
+export const CatalogueItemSchemaStep1 = (requestType: RequestType) => {
```

Comment:
```suggestion
export const CatalogueItemDetailsSchema = (requestType: RequestType) => {
```

For consistency since the step 2 is currently `PropertiesStepSchema`.